### ISSUE TITLE
Migration problems in Django 1.7

### ISSUE THREAD

**Issue by z0rr0** [futurecolors/django-geoip#41]
There is a problem during migrations MyCustomLocation model in Django 1.7 (RC2). I used embedded migration possibilities.

Following commands return successful result after a preparing of migration directory:

```
python manager.py makemigrations django_geoip
python manager.py migrate django_geoip
```

But there is a problem after a creation new model for [custom location](http://django-geoip.readthedocs.org/en/latest/usage.html#high-level-api-usage) in `customapp`, `makemigrations` command returns errors:

```
python manager.py makemigrations customapp
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
...
 File "/home/user/customapp/migrations/0006_customlocation.py", line 8, in <module>
    class Migration(migrations.Migration):
  File "/home/user/customapp/migrations/0006_customlocation.py", line 27, in Migration
    bases=(django.db.models.base.NewBase, models.Model),
AttributeError: 'module' object has no attribute 'NewBase'
```

I'm ready to prepare more details if it's needed. Can it be related with a new migration mechanism in Django 1.7?

**Comment by coagulant** [comment]
Can you provide more details? It might be app bug or a Django issue after all.

**Comment by z0rr0** [comment]
Hi,
I updated Django 1.7 and django-geoip to the latest versions and I only added `django_geoip` to `INSTALLED_APPS`, `migrate` command returns following error:

```
python manage.py migrate
Operations to perform:
  Synchronize unmigrated apps: widget_tweaks, debug_toolbar, django_geoip
  Apply all migrations: django_geoip, auth, sessions, contenttypes, admin
Synchronizing apps without migrations:
  Creating tables...
    Creating table django_geoip_country
    Creating table django_geoip_region
    Creating table django_geoip_city
    Creating table django_geoip_iprange
  Installing custom SQL...
  Installing indexes...
Running migrations:
  Applying django_geoip.0001_initial...Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/user/env/lib/python3.4/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/user/env/lib/python3.4/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/user/env/lib/python3.4/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/user/env/lib/python3.4/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/home/user/env/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 160, in handle
    executor.migrate(targets, plan, fake=options.get("fake", False))
  File "/home/user/env/lib/python3.4/site-packages/django/db/migrations/executor.py", line 63, in migrate
    self.apply_migration(migration, fake=fake)
  File "/home/user/env/lib/python3.4/site-packages/django/db/migrations/executor.py", line 91, in apply_migration
    if self.detect_soft_applied(migration):
  File "/home/user/env/lib/python3.4/site-packages/django/db/migrations/executor.py", line 135, in detect_soft_applied
    apps = project_state.render()
  File "/home/user/env/lib/python3.4/site-packages/django/db/migrations/state.py", line 57, in render
    self.apps = Apps([AppConfigStub(label) for label in sorted(self.real_apps + list(app_labels))])
  File "/home/user/env/lib/python3.4/site-packages/django/apps/registry.py", line 56, in __init__
    self.populate(installed_apps)
  File "/home/user/env/lib/python3.4/site-packages/django/apps/registry.py", line 89, in populate
    "duplicates: %s" % app_config.label)
django.core.exceptions.ImproperlyConfigured: Application labels aren't unique, duplicates: django_geoip
```

Tables `city, country, iprage, region` were created in the database. Possibly, the root cause can be related with https://docs.djangoproject.com/en/1.7/ref/applications/#configuring-applications

**Comment by coagulant** [comment]
This error `Application labels aren't unique, duplicates: django_geoip` might suggest you have misconfigured applications. Can you show your settings.py?

**Comment by coagulant** [comment]
Btw, thanks to your report I've fixed a packaging issue, please upgrade to 0.5.1 (might fix the issue, but not sure)

**Comment by z0rr0** [comment]
Hi @coagulant 
I have checked, all is Ok now. Thank you very much for the help.
